packages/wasm-parser/src/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import { traverse } from "@webassemblyjs/ast";
+const t = require("@webassemblyjs/ast");
```

Comment:
I forgot to warn you, since we have a cyclic dependency with some modules, JavaScript Module sometime doesn't work, but I guess you already figure that out. There's an issue that I need to fix about it #184